magprime/templates/regextra.html

```diff
@@ -24,15 +24,18 @@
             $('#guitar_keychain,#tax_exempt').empty();
         }
         {% if c.PAGE_PATH != '/registration/form' %}
+            // TODO: rip this hardcoded stuff out?
             if ($.field('amount_extra')) {
                 $.field('amount_extra').parents('.form-group').append($('#tax_exempt'));
```

Comment:
This is the div describing how MAGFest is a 501(c)(3) and that donations are tax-exempt. We'll want to keep this on the page, though we may want to change where we shove it (this line puts it right after the kick-in selections IIRC, append would put it inside the `.form-group` div)